index.html

```diff
@@ -1,19 +1,123 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body class="body">
+    <header class="header">
+      <div class="header-logo">
+        <a href="index.html" class="header-logo-link"
+          ><img src="img/karma-logo.svg" alt="" class="logo"
+        /></a>
+      </div>
+      <nav class="header-nav">
+        <a href="" class="header-nav-link">Meet Karma</a>
+        <a href="" class="header-nav-link">How It Works</a>
+        <a href="store.html" class="header-nav-link">Store</a>
+        <a href="" class="header-nav-link">Blog</a>
+        <a href="" class="header-nav-link">Help</a>
+        <a href="" class="header-nav-link">Login</a>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="section1">
+        <article class="section1-article">
+          <h1 class="section1-title">Introducing Karma</h1>
+          <h3 class="section1-text">Bring WiFi with you, everywhere you go.</h3>
+          <button class="section1-button">Learn More</button>
+        </article>
+      </section>
+      <section class="section2">
+        <h1 class="section2-title">Everyone needs a little Karma.</h1>
+        <div class="section2-articles">
+          <article class="section2-article1">
```

Comment:
I cannot see you using this class anywhere for styling. Is it necessary?
Wouldn't be better for to you the same class for the same elements (that's why we use classes instead of Ids)?

Reply:
Hi Saadia, you're right, I don't use the class "section2-article1" anywhere in the CSS. I am just in the habit of putting classes on most elements, even if they don't require them. Thanks for pointing it out, I will try to remember it in future. Thanks.